### PR TITLE
Move all of include/ into include/SViper/, so that other projects can…

### PR DESCRIPTION
**Pull request by joshuak94**
… do #include <SViper/sviper.h>. Also introduce SViper namespace.

### CODE REVIEW

**Comment by smehringer** [comment]
I made some fixes to the files. Can you rebase on current master? The Makefile does not exist anymore. I made this repo into a cmake based project. If you get into troubles I'll help

**Comment by joshuak94** [comment]
@smehringer Also done!

**Comment by smehringer** [comment]
Also rebase again :grin:

**Comment by joshuak94** [comment]
@smehringer Can this one and #10 be merged?

File: include/SViper/auxiliary.h [hunk 1]
```diff
@@ -1,12 +1,14 @@
 #pragma once
-#include <basics.h>
-#include <config.h>
+#include <SViper/basics.h>
```
Comment (smehringer) [inline include/SViper/auxiliary.h]: Can you change it to `sviper` (no capitals)?

**Review by smehringer — CHANGES_REQUESTED** [review]

**Comment by joshuak94** [comment]
@smehringer Renamed it! I had to remove the "sviper" line from .gitignore, but I think it's okay since the entire build subdirectory is ignored anyways.

**Comment by joshuak94** [comment]
@smehringer bump!

**Review by smehringer — APPROVED** [review]

**Pull request merged**